README.md

```diff
@@ -19,24 +19,46 @@ The gDR suite offers a full stack solution for storing, processing, and visualiz
 			| BumpyMatrix |			                       ---------------------
 			+-------------+   \           +----------+                +-----------+
 		+----------------------+    --------  | gDRcore  | <------------- | gDRimport | 
-		| SummarizedExperiment |  /           +----------+   load raw     +-----------+
+		| MultiAssayExperiment |  /           +----------+   load raw     +-----------+
 		+----------------------+                   :        files for analysis
                                                            :
-				      perform              :   gDRcore::runDrugResponseProcessingPipeline()
+				      perform              :   gDR::runDrugResponseProcessingPipeline()
 				      normalization, 	   :   
 				      averaging, 	   :   
 				      curve fitting	   : 
                                                            :
 
-                                                     SummarizedExperiment object
+                                                     MultiAssayExperiment object
```

Comment:
It's in fact `MultiAssayExperiment` with experiments being `SummarizedExperiment` objects with BumpyMatrix assays. But that's too complex a description. The current one is probably fair enough.

Reply:
Good point. I will change to more accurately describe.